File: passenger-service/src/main/java/com/software/modsen/passengerservice/service/PassengerRatingService.java
package com.software.modsen.passengerservice.service;

import com.software.modsen.passengerservice.entity.PassengerRating;

public interface PassengerRatingService {
    PassengerRating getByPassengerId(Long passengerId);
    PassengerRating save(PassengerRating passengerRating);
}

File: passenger-service/src/main/kotlin/com/software/modsen/passengerservice/service/PassengerRatingService.kt
package com.software.modsen.passengerservice.service

import com.software.modsen.passengerservice.entity.PassengerRating

interface PassengerRatingService {
    fun getByPassengerId(passengerId: Long): PassengerRating
    fun save(passengerRating: PassengerRating): PassengerRating
}